spec/models/item_spec.rb

```diff
@@ -9,47 +9,104 @@
   let(:drawing) { 'foo.jpg' }
   let(:sign) { instance_double('Sign', id: sign_id, gloss_maori: maori_name, gloss_main: name, drawing: drawing) }
 
```

Comment:
I think we tend to use lowercase at the start of it/context blocks because it looks nicer when the test case sentence gets concatenated together.
🐘

Reply:
Good point, will change :sparkles: